packages/firebase_ml_vision/test/image_labeler_test.dart

```diff
@@ -0,0 +1,165 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
The code below was moved to its own file from `firebase_ml_vision_test.dart` changes can be seen [here](https://github.com/flutter/plugins/pull/1364/commits/c9c42d8175dd30f21243d926165af466e887f590).